From: =?utf-8?q?Cristian_Rodr=C3=ADguez?= <crrodriguez@opensuse.org>
Date: Thu, 29 May 2014 14:17:37 -0400
Subject: tty-ask-password-agent: Do tell what directory we failed to open

(cherry picked from commit 267b3e41df5a2181f2911433539f81de2fa1511a)
(cherry picked from commit 54fcd1a651e89f4dd901dbecabf212f732006423)
---
 src/tty-ask-password-agent/tty-ask-password-agent.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tty-ask-password-agent/tty-ask-password-agent.c b/src/tty-ask-password-agent/tty-ask-password-agent.c
index cb74c72..1e9ad9e 100644
--- a/src/tty-ask-password-agent/tty-ask-password-agent.c
+++ b/src/tty-ask-password-agent/tty-ask-password-agent.c
@@ -503,7 +503,7 @@ static int show_passwords(void) {
                 if (errno == ENOENT)
                         return 0;
 
-                log_error("opendir(): %m");
+                log_error("opendir(/run/systemd/ask-password): %m");
                 return -errno;
         }
 
